conan/tools/cmake/presets.py

```diff
@@ -20,22 +20,22 @@ def _add_build_preset(conanfile, multiconfig):
 
 def _build_preset_name(conanfile):
     build_type = conanfile.settings.get_safe("build_type")
-    suffix = get_build_folder_vars_suffix(conanfile)
-    if suffix:
+    custom_conf = get_build_folder_custom_vars(conanfile)
+    if custom_conf:
         if build_type:
-            return "{}-{}".format(build_type, suffix)
+            return "{}-{}".format(custom_conf, build_type.lower())
         else:
-            return suffix
-    return build_type or "default"
+            return custom_conf
+    return build_type.lower() if build_type else "default"
 
 
 def _configure_preset_name(conanfile, multiconfig):
     build_type = conanfile.settings.get_safe("build_type")
-    suffix = get_build_folder_vars_suffix(conanfile)
-    base = "default" if multiconfig or not build_type else build_type
-    if suffix:
-        return "{}-{}".format(base, suffix)
-    return base
+    custom_conf = get_build_folder_custom_vars(conanfile)
+    suffix = "default" if multiconfig or not build_type else str(build_type).lower()
+    if custom_conf:
+        return "{}-{}".format(custom_conf, suffix)
```

Comment:
I would put `default` only if there is no `custom_conf`. It looks much better to have:
- config preset `gcc-11`
- build preset `gcc-11-release`

rather than:

- config preset `default-gcc-11`
- build preset `gcc-11-release`

In this case, what is the `default` needed for? I think that only `gcc-11` is much better.
Only when there is no `custom_conf` than we need to provide some name and `default` is as good as any other here in such case ;-)

Reply:
Good point. Done!